pkgs/applications/audio/renoise/default.nix

```
@@ -12,21 +12,31 @@ in
 
 stdenv.mkDerivation rec {
   pname = "renoise";
-  version = "3.3.2";
-
-  src =
-    if stdenv.hostPlatform.system == "x86_64-linux" then
-        if releasePath == null then
-        fetchurl {
-          urls = [
-              "https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_Linux.tar.gz"
-              "https://web.archive.org/web/https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_Linux.tar.gz"
-          ];
-          sha256 = "0d9pnrvs93d4bwbfqxwyr3lg3k6gnzmp81m95gglzwdzczxkw38k";
-        }
-        else
-          releasePath
-    else throw "Platform is not supported. Use installation native to your platform https://www.renoise.com/";
+  version = "3.4.3";
+
+  src = if stdenv.hostPlatform.system != "x86_64-linux" then
+    throw
+    "Platform is not supported. Use installation native to your platform https://www.renoise.com/"
+  else if releasePath != null then
+    releasePath
+  else
+    let
+      mkUrl = { urlStem, archSuffix }:
+        lib.strings.concatStrings [
+          urlStem
+          "https://files.renoise.com/demo/Renoise_"
+          (urlVersion version)
+          "_Demo_Linux"
+          archSuffix
+          ".tar.gz"
+        ];
+    in fetchurl {
+      urls = map mkUrl (lib.attrsets.cartesianProductOfSets {
+        urlStem = [ "" "https://web.archive.org/web/" ];
+        archSuffix = [ "_x86_64" "" ];
+      });
+      sha256 = "0nkyidxyp8r7jgdjld6f1bsk59927rw0n1n4j492ncwblwszmp0j";
```

```suggestion
      hash = "0nkyidxyp8r7jgdjld6f1bsk59927rw0n1n4j492ncwblwszmp0j";
```

you will also need to change has to SRI representation

---

Resolved in https://github.com/NixOS/nixpkgs/pull/271772/commits/51869a32f98f8a7d42e46f2e00259c014a24de8a.